paddle/fluid/operators/mkldnn/batch_norm_mkldnn_op.cc

```diff
@@ -144,7 +144,9 @@ class BatchNormMKLDNNOpKernel : public paddle::framework::OpKernel<T> {
 
     auto src_tz = paddle::framework::vectorize<int64_t>(x->dims());
     auto scale_tz = paddle::framework::vectorize<int64_t>(scale->dims());
-    PADDLE_ENFORCE(scale_tz.size() == 1, "Dims of scale tensor is NOT 1");
+    PADDLE_ENFORCE_EQ(
+        scale_tz.size(), 1,
+        platform::errors::InvalidArgument("Dims of scale tensor must be 1"));
```

Comment:
It is better to print the wrong size, such as:
```
PADDLE_ENFORCE_EQ(
        scale_tz.size(), 1,
        platform::errors::InvalidArgument("Dims of scale tensor must be 1. But received scale_tz's size is %s. ", scale_tz.size()));
```

Reply:
Done